src/frontend/src/lib/types/idb.ts

```diff
@@ -1,4 +1,10 @@
-import type { EthAddress } from '$lib/types/address';
+import type { BtcAddress, EthAddress } from '$lib/types/address';
+
+export interface IdbBtcAddress {
+	address: BtcAddress;
+	createdAtTimestamp: number;
```

Comment:
should we have a common interface `IdbAddress` that declares the timestamps and implements it for Btc and Eth?

Reply:
Yes! Agreed, something like:

```typescript
interface IdbAddress<T> {
	address: T;
	createdAtTimestamp: number;
	lastUsedTimestamp: number;
}

export type IdbBtcAddress = IdbAddress<BtcAddress>;

export type IdbEthAddress = IdbAddress<EthAddress>;
```

Reply:
Nice